crates/subspace-networking/src/node_runner.rs

```diff
@@ -457,6 +457,7 @@ impl NodeRunner {
             SwarmEvent::Behaviour(Event::Autonat(event)) => {
                 self.handle_autonat_event(event).await;
             }
+            // TODO: implement ExpiredListenAddr/ListenerClosed by removing the address from shared.listeners
```

Comment:
We always the upstream issue or pr link here as a reference so its easy to watch them.

Seems like the PR is merged, can we not fix the TODO already ?

Reply:
Happy to fix the TODO, maybe @nazar-pc can say if it needs fixing? Or how to fix it?